src/utils/stringCollector.ts

```diff
@@ -0,0 +1,26 @@
+export default class StringCollector {
+    private response: string = '';
+
+    process ( data: AllowSharedBufferSource ): Array<string> {
+        const decoder = new TextDecoder();
+
+        this.response += decoder.decode(data);
+
+        if ( this.response.length !== 0) {
+            const lastLineValid = this.response.endsWith('\r\n');
```

Comment:
```suggestion
            const isLastLineValid = this.response.endsWith('\r\n');
```

or even `isValidEnd` as we have only one line at this moment

Reply:
String collector removed from the request